pages/dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/configure-bastion/index.md

```diff
@@ -0,0 +1,127 @@
+
+1. Verify you have the SSH agent running on the local computer.
+
+    * On macOS, the SSH agent runs by default.
+    * On Linux, run the `ssh-agent` command and follow the prompts displayed to start the agent.
+
+1. Add the private key to the SSH agent.
+
+    * On **macOS**, run a command similar to the following:
+
+      ```bash
+      ssh-add -k ${HOME}/.ssh/id_rsa
+      ```
+
+    * On **Linux**, run a command similar to the following::
+
+      ```bash
+      ssh-add ${HOME}/.ssh/id_rsa
+      ```
+
+1. Verify that the SSH agent forwarding is configured properly by manually opening a secure shell session connection to one of the cluster nodes.
+
+    For example, log in by running a command similar to the following:
+
+    ```bash
+    ssh -A <host>
+    ```
+
+If you are able to connect to the remote host without providing a password, you have successfully configured SSH agent forwarding.
+
+### Configure the inventory file
+
+After you have configured and tested SSH agent forwarding, you can configure the appropriate bastion host settings in the Ansible inventory file.
+
+For example, configure the following bastion host settings in the `inventory.yaml` file:
+
+* `bastion.hosts` lists the hosts through which a proxy secure shell connection can be made.
+   A host from this list is selected randomly for each SSH connection.
+* `ansible_user` specifies the user account used to open the secure shell connection into the hosts listed as `bastion.hosts`.
+* `ansible_port` specifies the port used to open the secure shell connection into the hosts listed as `bastion.hosts`.
+
+The following example illustrates how to configure these settings in the `inventory.yaml` file:
+
+```yaml
+control-plane:
+  hosts:
+    10.0.194.142:
+      ansible_host: 10.0.194.142
+    10.0.198.130:
+      ansible_host: 10.0.198.130
+    10.0.200.148:
+      ansible_host: 10.0.200.148
+
+node:
+  hosts:
+    10.0.130.168:
+      ansible_host: 10.0.130.168
+      node_pool: worker
+    10.0.133.221:
+      ansible_host: 10.0.133.221
+      node_pool: worker
+    10.0.139.120:
+      ansible_host: 10.0.139.120
+      node_pool: worker
+
+bastion:
+  hosts:
+    10.0.131.50:
+      ansible_host: 10.0.131.50
+  vars:
+    ansible_user: "centos"
+    ansible_port: 22
+
+all:
+  vars:
+    version: v1beta1
+    order: sorted
+    ansible_user: "centos"
+    ansible_port: 22
+```
+
+Note that you must specify the `ansible_user` and `ansible_port`.
+These settings define the user name and port number that the bastion host uses to establish the SSH connection between the bastion host and the cluster host.
+
+In editing the `inventory.yaml` file, you should also keep in mind that SSH agent forwarding is required and, therefore, you **must not** specify any value for the `ansible_ssh_private_key_file` setting.
+When you run `dkp up`, DKP validates that a valid private key has been loaded in the SSH agent for the provided public key.
```

Comment:
dkp up isn't the same - I think that this would either be dkp create cluster preprovisioned --flags --flags --flags

Or, it's probably going to be somethign where you kubectl apply -f your-cluster-yaml.yaml or something like that.

@joejulian @dkoshkin ?